### PR TITLE
Feature/avatar message pop up

### PR DESCRIPTION
**Pull request by JaHajRF**
avatar pop up messages added

### CODE REVIEW

**Review by kowalski — CHANGES_REQUESTED** [review]

File: packages/roomSync/src/services/config/protonsRequestMessage.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ message GetInitState {
   required bytes positionY = 5;
   required bytes userPeerId = 6;
   required bytes isDancing = 7;
+  required bytes shouldDisplayMessagePopUp = 8;
```
[the remark this answers is not in the document]
Comment (JaHajRF) [reply packages/roomSync/src/services/config/protonsRequestMessage.ts:60]: I'm still working on changes that were discussed during last call. This prop will be deleted in next commit.

[the remark this answers is not in the document]
Comment (kowalski) [reply packages/roomSync/src/services/config/protonsRequestMessage.ts:60]: ok, great, I got confused as I'm doing few reviews at the same time ;p

File: packages/webapp/src/components/Chat/Chat.stories.tsx [hunk 2]
```diff
@@ -18,9 +19,9 @@ storiesOf('components/Chat', module).add('default', () => {
     return { payload: color, type: 'setUserNameColor' }
   }
   const messages = [
-    { from: { avatarId: '1', color: 'red', userName: 'Pete' }, text: 'Hello there!' },
-    { from: { avatarId: '1', color: 'blue', userName: 'Mike' }, text: 'great song' },
-  ]
+    { from: { avatarId: '1', color: 'red', peerId: '1', userName: 'Pete' }, text: 'Hello there!' },
+    { from: { avatarId: '1', color: 'blue', peerId: '2', userName: 'Mike' }, text: 'great song' },
+  ] as Message[]
```
Comment (kowalski) [inline packages/webapp/src/components/Chat/Chat.stories.tsx:24]: prefer:

```
const messages: Messages[] = [...]
```
without as

File: packages/webapp/src/components/DancefloorAvatar/DancefloorAvatar.styles.ts [hunk 3]
```diff
@@ -31,3 +64,66 @@ export const BodyBack = styled.img`
...
+
+// export const MessagePopUpElement = styled.div`
+//   bottom: 4.3vw;
+//   left: 50%;
+//   position: absolute;
+//   z-index: 1;
+//   ${animateIn}
+//   ${animateOut}
+// `
+// export const MessageBox = styled.div`
```
Comment (kowalski) [inline packages/webapp/src/components/DancefloorAvatar/DancefloorAvatar.styles.ts:76]: what the deal with all these commented out code?

File: packages/webapp/src/store/chatMessages/chatMessages.reducer.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+export const chatMessagesActions = {
+  addMessage: createAction<Message>(ChatMessages.ADD_MESSAGE),
+  removeMessage: createAction<string>(ChatMessages.REMOVE_MESSAGE),
+}
+
+export const chatMessagesReducer = createReducer(initialState as ChatMessagesStore, (builder) => {
+  builder
+    .addCase(chatMessagesActions.addMessage, (state, action: PayloadAction<Message>) => {
+      state.usersLastMessages.delete(action.payload.from.peerId)
+      state.usersLastMessages.set(action.payload.from.peerId, action.payload)
```
Comment (kowalski) [inline packages/webapp/src/store/chatMessages/chatMessages.reducer.ts:23]: no need to delete, than set; set overwrite previous value, right?

File: packages/webapp/src/store/chatMessages/chatMessages.saga.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    from: {
+      avatarId: action.payload.avatarId,
+      color: action.payload.color,
+      peerId: action.payload.from,
+      userName: action.payload.userName,
+    },
+    text: action.payload.message,
+  } as Message
+  yield put(chatMessagesActions.addMessage(message))
+  yield delay(4000)
```
Comment (kowalski) [inline packages/webapp/src/store/chatMessages/chatMessages.saga.ts:19]: magic number, move to const

**Review by kowalski — CHANGES_REQUESTED** [review]

**Review by kowalski — APPROVED** [review]

**Pull request merged**